### ISSUE TITLE
Confusing sample IDs with Matrix Export(.csv)

### ISSUE THREAD

**Issue by berrytf** [ADAP-BIG/adap-big.github.io#5]
When exporting data as a table sample IDs are preserved in the output, however when exporting data as a matrix the samples you get sample names that are unclear and not connected to sample names. For example sample named 'POS_HDCDV303 Filter.mzML' becomes 'org.dulab.adapbig.base.Sample@43414f6a.' 

You can match the area data in matrix output to the table output to determine which sample is which, but it seems like an extra step that shouldn't be needed.

**Comment by asmirn1** [comment]
@berrytf Yes, I've noticed that also and made a fix for this bug. It'll be added to the next release.

**Comment by berrytf** [comment]
Thanks a ton. Great software by the way!

**Comment by asmirn1** [comment]
@berrytf Thank you so much!

Let me know if you notice other issues or have any comments/suggestions.
